tasks/src/main/java/ru/sberbank/school/task02/calculator/QuotePrice.java

```diff
@@ -0,0 +1,47 @@
+package ru.sberbank.school.task02.calculator;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+@Getter
+public class QuotePrice {
+    private BigDecimal amount;
+    private BigDecimal price;
+    @Setter
+    private BigDecimal lowerLimit;
+
+    public QuotePrice(BigDecimal amountInCurr, BigDecimal price) {
+        this.amount = amountInCurr;
+        this.price = price;
+    }
+
+    public BigDecimal getUpperLimit() {
+        return amount;
+    }
+
+    public BigDecimal getPricePerPiece() {
+        return BigDecimal.ONE.divide(price, 10, RoundingMode.HALF_UP);
+    }
+
+    @Override
+    public String toString() {
+        return  "Price " + price + " amount " + amount + " PricePerPiece: " + getPricePerPiece();
+    }
+
+    private boolean moreThenLowerLimit (BigDecimal amount) {
+        return amount.compareTo(lowerLimit) > 0;
+    }
+
+    private boolean lessThenUpperLimit (BigDecimal amount) {
```

Comment:
&#39;(&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1878376795>*